src/components/interaction-designer/Block.vue

```diff
@@ -250,6 +264,10 @@ export default {
       return !!filter(activeConnectionsContext, (context) => context.sourceId === block.uuid || context.targetId === block.uuid).length
     },
 
+    isBlockSelected() {
+      return this.block.vendor_metadata.io_viamo.uiData.isSelected
```

Comment:
This seems weird... Now I think about it, it's not really viamo specific to even store the block position in io_viamo as it's an integral part of the builder

I suppose this isn't something we want to think too deeply about right now though. It also seems a bit weird to me that if you export and import the flow it could already have blocks selected but it's not necessarily a bad thing... Happy to leave this right now.

Reply:
@jcbashdown , I agree on the fact that block positions are not `io_viamo`, maybe we decided to do so because viamo blocks persist position using this field id?

About new introcuded field, let me see if I can move it out of `io_viamo`.